07-documenting_R.Rmd

```diff
@@ -98,10 +96,14 @@ Our dev guide workflow uses continuous integration and deployment so changes to
 
 ## Instructions for reporting the CRAN policy bugs -- discussion in slack (random channel)
 
+If you find some typo or problem on [CRAN](https://cran.r-project.org) after checking the problem or typo you can write a polite email to `cran-sysadmin@r-project.org` and one of the R-core members working with CRAN. You'll probably get a prompt reply about how the issue is going to be fixed.
+
 **Note**:
 
 There is a `#core-documentation` channel on the [R Contributors slack](https://r-contributors.slack.com/) where you can discuss about the patches for improvements to R's documentation. 
 
 ## See also
 
 1. [Writing R documentation files](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Writing-R-documentation-files)
+2. [R manuals](https://cran.r-project.org/manuals.html): These are a part of the [R sources](https://github.com/r-devel/r-svn/tree/master/doc/manual). Hence, bug reports/patches can also be submitted via Bugzilla, e.g. https://bugs.r-project.org/bugzilla/show_bug.cgi?id=15221. Note that they are typically referred to by their file names "R-admin", "R-exts" etc. The manuals are written in [Texinfo](https://www.gnu.org/software/texinfo/) (file extension `.texi`). The manual should be referred to for [how to mark up text](https://www.gnu.org/software/texinfo/manual/texinfo/texinfo.html). 
+R-admin contains information on how to [build the manuals](https://cran.r-project.org/doc/manuals/r-release/R-admin.html#Making-the-manuals) - it would be a good idea to check the manuals build if you are providing a patch. But simple suggestions could be made by describing the change or providing suggested `.texi` marked-up text directly, e.g. https://bugs.r-project.org/bugzilla/show_bug.cgi?id=15193.
```

Comment:
Again - use hyperlink with bug title vs URL.

Ideally, someone would be able to build the manuals after following our instructions to build R from source. This probably requires an update to the installation instructions first, then perhaps we can add simple instructions here for building the manuals (cc @llrs) - no need to wait for this before merging this PR.

Reply:
I think that one doesn't need to build R from source to build the manuals, but I will check it.